mitodl/ocw-studio
# Revert social-auth-app-django to 5.4.0

### What are the relevant tickets?
No ticket - trying to resolve 'lxml & xmlsec libxml2 library version mismatch' error in RC
in RC, we currently have `xmlsec==1.3.14` whereas in production we have `xmlsec==1.3.13`. This is just an attempt to see if RC was broken by this change.

### How can this be tested?
docker-compose build
docker-compose run
Everything works as it should, django admin should work as expected.

### Additional Context
